src/node_crypto.cc

```diff
@@ -2590,10 +2591,15 @@ void CipherBase::Init(const char* cipher_type,
                                1,
                                key,
                                iv);
+  CHECK_NE(key_len, 0);
 
   ctx_.reset(EVP_CIPHER_CTX_new());
   const bool encrypt = (kind_ == kCipher);
-  EVP_CipherInit_ex(ctx_.get(), cipher, nullptr, nullptr, nullptr, encrypt);
+  if (1 != EVP_CipherInit_ex(ctx_.get(), cipher, nullptr,
```

Comment:
Style nit, but shouldn't this be done the other way around? Eg: `EVP_CipherInit_ex(...) != 1`? Would love to hear why you chose this over the former.

Reply:
Didn't know we had a preference. I usually do the opposite, but I think having `!= 1` at the end of a multi-line expression looks weird.

Fun fact, this style was commonly used to prevent accidental assignment of variables / pointers in boolean expressions.

Reply:
Because people would accidentally type `=` instead of `==` all the times? Sounds more likely in the era of butterfly keyboards, heh. If it makes sense for you to keep this, feel free to do so. I agree wholeheartedly that `!= 1` looks weird when in a multiline statement, just wanted to make sure we're not going against the style guide here.

Reply:
Yes, even though I personally still prefer `var == constant`, the only reason here is that it looks weird. If no one has a strong opinion against this, I'd prefer to keep it this way.